tests/storages_tests/journal_tests/test_journal.py

```diff
@@ -244,3 +258,18 @@ def test_raise_error_for_deprecated_class_import_from_journal() -> None:
         journal.JournalRedisStorage  # type: ignore[attr-defined]
     with pytest.raises(AttributeError):
         journal.BaseJournalLogStorage  # type: ignore[attr-defined]
+
+
+@pytest.mark.parametrize(
+    "log_storage_type,grace_period",
+    [
+        ("file_with_open_lock", 0),
+        ("file_with_open_lock", -1),
+        ("file_with_link_lock", 0),
+        ("file_with_link_lock", -1),
+    ],
+)
```

Comment:
It can be written more simply.

```suggestion
@pytest.mark.parametrize("log_storage_type", ("file_with_open_lock", "file_with_link_lock"))
@pytest.mark.parametrize("grace_period", (0, -1))
```